test/JDBC/input/BABEL-730-vu-cleanup.sql

```diff
@@ -0,0 +1,39 @@
+use test_babel730
+go
```

Comment:
We should add this to mvu schedule files

Reply:
Solved